python/mxnet/numpy/multiarray.py

```diff
@@ -6419,3 +6419,39 @@ def einsum(*operands, **kwargs):
     ...     np.einsum('ijk,ilm,njm,nlk,abc->',a,a,a,a,a, optimize=True)
     """
     return _mx_nd_np.einsum(*operands, **kwargs)
+
+@set_module('mxnet.numpy')
+def column_stack(tup):
+    """ column_stack(*args, **kwargs)
+
+    Stack 1-D arrays as columns into a 2-D array.
+
+    Take a sequence of 1-D arrays and stack them as columns
+    to make a single 2-D array. 2-D arrays are stacked as-is,
+    just like with `hstack`.  1-D arrays are turned into 2-D columns
+    first.
+
+    Parameters
+    ----------
+    tup : sequence of 1-D or 2-D arrays.
+        Arrays to stack. All of them must have the same first dimension.
+
+    Returns
+    -------
+    stacked : 2-D array
+        The array formed by stacking the given arrays.
+
+    See Also
+    --------
+    stack, hstack, vstack, concatenate
+
+    Examples
+    --------
+    >>> a = np.array((1,2,3))
+    >>> b = np.array((2,3,4))
+    >>> np.column_stack((a,b))
+    array([[1, 2],
```

Comment:
Notice the alignment
```python
array([[1, 2],
       [2, 3],
       [3, 4]])
```

Reply:
I didn't quite get it. Could you elaborate?

Reply:
I guess the code was outdated, examples in the doc string are aligned in the last commit.